Assets/Scripts/Widgets/PlaneStencil.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright 2020 The Tilt Brush Authors
```

Comment:
Please can this be attributed as such: `Copyright 2021 The Open Brush Authors`

Reply:
Sorry to be a pain, it should be Open Brush not Tilt Brush! 😅

Reply:
No worries, it's one of my first PR's so I appreciate your patience.